app/views/admin/badges/index.html.erb

```diff
@@ -1,5 +1,8 @@
 <header class="flex mb-6">
-  <h2 class="crayons-title">Badges</h2>
+  <!-- @ridhwana: temporary deuplicate header removed. Will remove this with old navigation code -->
```

Comment:
tiny typo in `deuplicate`

Reply:
lol i need to always check my comments for when my fingers type faster than my brain😅 Thanks ✅